### ISSUE TITLE
Clarity for 1.3.3: Sensory Characteristics needed

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1532]
https://www.w3.org/WAI/WCAG22/Understanding/sensory-characteristics

First I'm not sure if this SC is about:
an '"**instruction**" to get to a component / place' (a description **in** the web content) **OR** if the component / place on screen itself '**is**' the instruction (or does it apply for both?!)

Example: Is it the instruction to 'go to the right arrow and activate it to proceed' OR 'the right arrow IS the instruction' (indicating to go forward...?!)

First sentence of intent is:

> The intent of this Success Criterion is to ensure that all users can access instructions for using the content

The first example talks about:

> The cell corresponding to the time of a particular event has a specific background color and diamond shaped glyph so it can be identified by color and shape.

A blind user with a screen reader will NOT get the color instruction and also NOT the diamond glyph...!!

This is supported by the last sentence:

> WCAG was designed to apply only to controls that were displayed on a web page. The intent was to avoid describing controls solely via references to visual or auditory cues.

The first example is "describing controls solely via references to visual cues." (although it fits "**do not rely solely on color or shape**"

Then the next example is:

> The arrow is clearly labeled with "Next" and the instructions state, "To move to the next section of the survey, select the green arrow icon labeled 'Next' in the lower right corner below the last survey question." This example uses both positioning, color and labeling to help identify the icon.

Using the word "instructions" here, and not having a definition for instructions, suggests that the SC is all about a accompanying text "instructing on the use of a component, or alike" NOT about the component itself...?!

Is in this case the "green arrow icon labeled 'Next'" not already enough?

-----------------------

In short:

1. what is the definition of instruction here? (describing text of a component OR it can be the component itself (or color to convey meaning))
2. If you only have an icon of an arrow (left/right)  to go to the previous / next page, will this fail? (no label present BUT has an ACC name)

**Comment by patrickhlauke** [comment]
x-ref https://github.com/w3c/wcag/pull/767 (which I hope at some point will be merged, as it's been sitting there forever)

**Comment by bruce-usab** [comment]
Yes, there is a bit of a disconnect because 1.3.3 is scoped strictly to _instructions_ and _only_ instructions.

1.4.1 is _much_ broader.

In my experience, this has worked out okay because 1.3.1 covers so much ground.  But 1.3.3 only being applicable to instructions trips reviewers up all the time.  This is something that definitely leaves room for improvement, but maybe not until 3.0?

**Comment by patrickhlauke** [comment]
> But 1.3.3 only being applicable to instructions trips reviewers up all the time

then maybe we can push for some more clarity ... for instance accepting PR #767 which has been dormant and seems fairly non-controversial (i'm biased, of course)

**Comment by alastc** [comment]
@jake-abma does #767 address this sufficiently?

**Comment by alastc** [comment]
> Using the word "instructions" here, and not having a definition for instructions, suggests that the SC is all about a accompanying text "instructing on the use of a component, or alike" NOT about the component itself...?!

The instructions is about the accompanying text, not the component. How about finishing the example with:

> This example uses positioning and color, with labeling to also identify the icon and pass this criterion. 

That's in #1571 

I think with #767 (merged) that should close this issue?

**Comment by alastc** [comment]
Discussed at the meeting today: https://www.w3.org/2021/01/05-ag-minutes.html#t08

**Comment by jake-abma** [comment]
not sure if all issues in this issue are fixed yet, we only discussed the second example yesterday... leave open for now till I've inspected the other issues mentioned.

**Comment by alastc** [comment]
Hi Jake,

We discussed the others previously, check the latest version as both examples have been updated: https://w3c.github.io/wcag/understanding/sensory-characteristics.html

Cheers,

-Alastair

**Comment by jake-abma** [comment]
ok, missed that one, thx @alastc !
